src/lib/Timeline.js

```diff
@@ -81,6 +81,7 @@ export default class ReactCalendarTimeline extends Component {
     minResizeWidth: PropTypes.number,
     fixedHeader: PropTypes.oneOf(['fixed', 'sticky', 'none']),
     stickyOffset: PropTypes.number,
+    scrollableContainer: PropTypes.object.isRequired,
```

Comment:
Make this propType not required since we have a default value in defaultProps.

Reply:
1. React checks propTypes after it defaults `props` with `defaultProps`
2. We declare `scrollableContainer: window` as default prop
3. If we relax requirement to `non-required object`, then it will be possible to pass `scrollableContainer={null}`, component will fail and this will emit no validation warnings.

Now it works correctly with no false positive react propTypes warnings